### ISSUE TITLE
[0.10.0-rc1] "show measurements" yields i/o timeout

### ISSUE THREAD

**Issue by dmke** [influxdata/influxdb#5506]
Since 0.10.0-rc1 (on Debian Jessie/8.3 using the official downloads), I get the following output on `SHOW MEASUREMENTS`:

```
ERR: read message size: read tcp 127.0.0.1:8088: i/o timeout
```

(here's the [relevant snippet from the logs](https://gist.github.com/dmke/b679ea429cb74a640808))

This error is also present in 0.10.0.n1454400035. Using 0.10.0-beta2 it yields the desired result.

The database was created and filled with rc1 (so I guess, TSM?), over which beta2 was installed, over which the nightly was installed. Please let me know if you need a dataset to reproduce this, or any other information.

**Comment by rossmcdonald** [comment]
@dmke Can you paste the output from a `SHOW SERVERS` command run on this instance? And was this instance upgraded from 0.9?

**Comment by dmke** [comment]
No, the instance wasn't upgraded.

```
$ influxd version
InfluxDB v0.10.0 (git: 0.10.0 58e0eed9cb166e0b004e6b41a4d64c39d24b40f4, built 2016-01-23T02:21:10.382055)

$ influx -username root -password root -execute 'show servers'
name: data_nodes
----------------
id	http_addr	tcp_addr
2	localhost:8088	localhost:8088
3	localhost:8086	localhost:8088


name: meta_nodes
----------------
id	http_addr	tcp_addr
1	localhost:8091	localhost:8088
```

I also have git-bisected the source and might have found the culprit in the commit 31c2e7012 (@corylanou). I don't know wether or not `go clean './...' && go install './...'` has created the right binaries though (`$GOPATH/bin/influxd version` reported to be v0.9)

**Comment by rossmcdonald** [comment]
@dmke Yep, that was the commit that should have fixed this. I've never seen this occur on an instance that wasn't upgraded, though.

What are the contents of your `node.json` file? It should be under `~/.influxdb/meta` I believe if you're not using a packaged install (or `/var/lib/influxdb/meta` for a packaged install).

**Comment by dmke** [comment]
Just this: `{"ID":1,"MetaServers":[":8091"]}`

**Comment by dmke** [comment]
Addendum: There are two `node.json` files:

```
$ cat /var/lib/influxdb/meta/node.json 
{"ID":1,"MetaServers":[":8091"]}
$ cat /var/lib/influxdb/node.json 
{"ID":3,"MetaServers":["localhost:8091"]}
```

**Comment by rossmcdonald** [comment]
@dmke What is the configuration set to for the `meta.dir` variable? Also, is there a globally scoped `dir` variable in your config?

**Comment by dmke** [comment]
My configuration does not differ from the current [config.sample.toml](https://github.com/influxdata/influxdb/blob/c9d82ad/etc/config.sample.toml):

- `dir = "/var/lib/influxdb"`
- `meta.dir = "/var/lib/influxdb/meta"`

**Comment by dmke** [comment]
Hm. Out of curiosity I've purged the package, nuked `/var/lib/influxdb`, reinstalled the rc1 package and imported my dataset again. Now I can't reproduce the error anymore; `show measurements` yields the right results.

Tomorrow, I will try to import the dataset with beta2 and then upgrade to rc1... maybe that reproduces the error (if not, I'll just close this issue).

**Comment by jwilder** [comment]
@dmke It looks like you ran into an issues we had with upgrades that showed up in `beta2`.  The top-level `dir` config option was just removed to help prevent this from happening again in #5515 as well as fixes from #5447 which fixes different issues that you also ran into.

Going to close this since this should be addressed by `rc1` and occurred frequently w/ `beta2` build.

**Comment by dmke** [comment]
Roger that. Thank you both for your support!
